blocks/explore-cards/explore-cards.css

```diff
@@ -1 +1,154 @@
-/* TBD */
\ No newline at end of file
+.explore-cards-wrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.explore-cards {
+    width: 100%;
+}
+
+.story-type p {
+    font-size: var(--body-font-size-xs);
+    font-weight: 700;
+    line-height: 13.2px;
+    letter-spacing: 0.06em;
+}
+
+.card-title p {
+    font-size: var(--heading-font-size-xs);
+    font-weight: 700;
+    line-height: 24px;
+}
+
+.explore-cards .heading {
+    margin-bottom: var(--spacing-20);
+}
+
+.explore-cards  .heading p {
+    font-size: var(--heading-font-size-m);
+    line-height: 30.8px;
+}
+
+.explore-card-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: var(--spacing-20);
+    width: calc(100vw - 68px);
+    overflow-x: scroll;
+}
+
+.explore-card {
+    position: relative;
+    border-radius: 10px;
+    height: 400px;
```

Comment:
hardcoding height is not a great idea. min-height perhaps if it is really required?